scripts/sync_world_map_langs.py

```
@@ -0,0 +1,522 @@
+    """langs: {'zh': data, 'en': data, ...} -> (name_map, catalog_map, levels)"""
+    # name_map: zh -> {en, ja, ko}  (仅同构层级/site)
+    # catalog_map: subType id -> {zh, en, ja, ko}
+    # levels: 每 map/level 的 site 列表（用于 es 匹配与快照）
+    zh_data = langs["zh"]
+
+    name_map = {}
+    catalog_map = {}
+    levels = []
+
+    for i, m_zh in enumerate(zh_data["maps"]):
+        m = {l: langs[l]["maps"][i] for l in langs}
```

Ruff (0.16.0)

[error] Ambiguous variable name: `l`

(E741)